cd/scripts/release-controller.sh

```diff
@@ -122,6 +122,8 @@ if ! (echo "$VERSION" | grep -Eq "stable$"); then
   DEFAULT_DOCKER_REPOSITORY="public.ecr.aws/aws-controllers-k8s/$AWS_SERVICE-controller"
   DOCKER_REPOSITORY=${DOCKER_REPOSITORY:-"$DEFAULT_DOCKER_REPOSITORY"}
 
+  ensure_repository "$AWS_SERVICE" "$AWS_SERVICE-controller"
```

Comment:
how about changing this to `ensure_repository "$AWS_SERVICE"`?

and then inside that method you ensure both "$AWS_SERVICE"-controller and "$AWS_SERVICE"-chart repositories.

This way you can make sure both the repositories are present before building controller image.